src/Language/Haskell/Liquid/Types.hs

```diff
@@ -458,8 +458,9 @@ instance Hashable (PVar a) where
 ------ Strictness --------------------------------------------------
 --------------------------------------------------------------------
 
-instance NFData Var
-instance NFData SrcSpan
+instance NFData Var where rnf _ = ()
```

Comment:
IF we have to choose between two GHC versions and Werrors, the Werrors flag
is more useful, especially in long term.
On Mar 27, 2015 3:32 PM, "Eric Seidel" <notifications@github.com> wrote:

> In src/Language/Haskell/Liquid/Types.hs
> <https://github.com/ucsd-progsys/liquidhaskell/pull/364#discussion_r27335842>
> :
>
> > @@ -458,8 +458,9 @@ instance Hashable (PVar a) where
> >  ------ Strictness --------------------------------------------------
> >  --------------------------------------------------------------------
> >
> > -instance NFData Var
> > -instance NFData SrcSpan
> > +instance NFData Var where rnf _ = ()
>
> Unfortunately not, because standalone deriving requires the data
> constructors to be in scope, which they are not (at least for Var).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ucsd-progsys/liquidhaskell/pull/364/files#r27335842>.
>